apps/prs/src/app/pr-table/columns/status/status.component.ts

```diff
@@ -0,0 +1,26 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {BaseColumn} from '../base';
+
+@Component({
+  selector: 'status-column',
+  templateUrl: './status.component.html',
+  styleUrls: ['./status.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class StatusColumn extends BaseColumn {
+  name = 'status';
+
+  getStatusIconForStatus(status: string) {
```

Comment:
Could we use an octokit type here for the status values? 

Also nit: I think we should not throw errors without message. Obviously the stack trace could always help, or source-maps, but a little message is low-effort to add.